### DISCUSSION TITLE
money crashing game

### DISCUSSION THREAD

**Discussion by iibus1** [MAFINS/MenyooSP#511] [Q&A]
hi is this happening to anyone else or just me when i try to give franklin unlimited money it just crashes my game so I'm wondering if this is happening to anyone else and can you/when you update the mod can you fix this bug 

thanks iibus1

**Comment by BSK-YouTube** [comment]
Have you tried installing these files and running the game with them?
----------------------------------------------------------------------------
Script Hook V
Script Hook V Dot Net
Pack file Limit Adjuster
Heap Adjuster
Gameconfig

**Reply by BSK-YouTube** [reply]
I'm not sure but maybe you have some other mods and they conflict, it's worth checking before saying that the game crashes because of menyoo

**Reply by iibus1** [reply]
Ok I will give it a try.
